src/System.IO.FileSystem/src/System/IO/Enumeration/FileSystemEnumerableFactory.cs

```diff
@@ -20,11 +20,17 @@ internal static class FileSystemEnumerableFactory
         // which is not true in Windows and as such we'll escape any that occur on the input string.
         private readonly static char[] s_unixEscapeChars = { '\\', '"', '<', '>' };
 
-        internal static void NormalizeInputs(ref string directory, ref string expression, EnumerationOptions options)
+        internal static bool NormalizeInputs(ref string directory, ref string expression, EnumerationOptions options)
```

Comment:
This could use some xml comments now, particularly as we're returning an unnamed bool.

Reply:
Done.